bazel/src/main/java/com/google/api/codegen/bazel/BUILD.bazel.raw_api.mustache

```diff
@@ -113,6 +114,13 @@ php_grpc_library(
     deps = [":{{name}}_php_proto"],
 )
 
+php_gapic_assembly_pkg(
+    name = "{{type_only_assmebly_name}}-php",
```

Comment:
nit: `assmebly` - is this how it's called in the template engine?

Reply:
good catch! Copy+pasta == foot gun.

Fixed!